wme-address-point-helper.user.js

```diff
@@ -299,10 +362,12 @@
     createPoint(true)
   }
 
+	// 2. By disabling this check, the script allows users to create a copied new point even if the selected landmark does not have a house number.
```

Comment:
maybe better to make this behaviour optional? otherwise I don't see a reason to keep the dead code in the script.